### ISSUE TITLE
Bug: Login and sign up page get compress in different resolution 

### ISSUE THREAD

**Issue by nlok5923** [anitab-org/open-source-programs-web#105]
### Describe the bug
Login and register page compresses in different resolutions probably when resolution is less than 480 X 671

### To Reproduce
1. Go to Landing page
2. Resize login and register page

### Expected behavior
It should hold a flexbox property that login and register component should acquire the center upper half or lower upper half area of landing page 

### Screenshots
![image](https://user-images.githubusercontent.com/56120084/109963492-7b830700-7d12-11eb-8107-fdc081da6b93.png)
![image](https://user-images.githubusercontent.com/56120084/109963573-95244e80-7d12-11eb-92b3-077bd76f57b6.png)

### Desktop (please complete the following information):
 - OS: [e.g. iOS] All
 - Browser [e.g. chrome, safari] All

### Smartphone (please complete the following information):
 - Device: [e.g. iPhone6] All
 - OS: [e.g. iOS8.1] All
 - Browser [e.g. stock browser, safari] All

### Additional context
login and register page should be clearly accessible by user

**Comment by codesankalp** [comment]
Yes, @nlok5923 this is a valid UI issue. I was also thinking to open it in the future.
Also, In mobile view, it is not responsive and looks bad.
![image](https://user-images.githubusercontent.com/56037184/109979799-49c76b80-7d25-11eb-9754-a5bb8bd0ae36.png)

**Comment by ruddi10** [comment]
@codesankalp @keshakaneria  can I work on this

**Comment by codesankalp** [comment]
@ruddi10 As you are assigned to an issue, I can't assign you.

**Comment by nlok5923** [comment]
@codesankalp can u please assign this to me.

**Comment by codesankalp** [comment]
Sure @nlok5923, Assigning you.

**Comment by nlok5923** [comment]
@codesankalp what type of changes you want me to do . or shall i hide the anitaborg logo at lower resolutions

**Comment by codesankalp** [comment]
@nlok5923 You can change the UI according to scaling i.e. use transforms to change UI in mobile or any other method such that mobile and desktop UI looks good.

**Comment by codesankalp** [comment]
Any progress @nlok5923?

**Comment by codesankalp** [comment]
Unassigning @nlok5923 due to inactivity.

**Comment by Aryamanz29** [comment]
I want to give it a try, Could you please assign it to me @codesankalp

**Comment by Aryamanz29** [comment]
Can we use material UI `useMediaQuery `for this?
ref : https://material-ui.com/components/use-media-query/
@codesankalp

**Comment by atharvashirdhankar** [comment]
@codesankalp can I work on this issue .
It's Just a front end work I guess.
 I'll do it.

**Comment by codesankalp** [comment]
Assigning you @StarTrooper08.
Yes, it's just related to the frontend, but please make sure that the login feature doesn't break.

**Comment by atharvashirdhankar** [comment]
> Assigning you @StarTrooper08.
> Yes, it's just related to the frontend, but please make sure that the login feature doesn't break.

Yes , sure👍

**Comment by atharvashirdhankar** [comment]
Hey sorry for the delay to solve this issue.
I forgot to ask that in which file I have to make changes 😅.

**Comment by codesankalp** [comment]
@StarTrooper08 First try to explore the project.
You have to make changes in the [Login](https://github.com/anitab-org/anitab-forms-web/blob/develop/src/components/Login.js) and Signup Component.

**Comment by atharvashirdhankar** [comment]
Okay

**Comment by vj-codes** [comment]
@StarTrooper08 any updates?

**Comment by pankajsahu221** [comment]
@codesankalp I can work on this issue, please assign it to me.

**Comment by pankajsahu221** [comment]
@codesankalp I can make the login and register page fully responsive using pure CSS without breaking any feature. If you need any other changes, then do let me know. Please assign the task to me, I would love to work on it.

**Comment by codesankalp** [comment]
> @codesankalp I can make the login and register page fully responsive using pure CSS without breaking any feature. If you need any other changes, then do let me know. Please assign the task to me, I would love to work on it.

Instead of using pure CSS try to use react-bootstrap as much as possible.

**Comment by pankajsahu221** [comment]
> > @codesankalp I can make the login and register page fully responsive using pure CSS without breaking any feature. If you need any other changes, then do let me know. Please assign the task to me, I would love to work on it.
> 
> Instead of using pure CSS try to use react-bootstrap as much as possible.

Okay!
